src/status_im/ui/screens/profile/user/views.cljs

```diff
@@ -212,9 +216,11 @@
 
           ;; flatlist contents
           header               (header multiaccount)
+          keycard-account?   (boolean (get-in multiaccounts [address :keycard-key-uid]))
```

Comment:
is it mandatory to cast to boolean here?

Reply:
@yenda not mandatory as this value could be either truthy string or nil